### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by SergeiMikhailovskii**

### CODE REVIEW

File: app/src/main/java/asus/example/com/mvpexample/IAddDataEvents.java [hunk 1]
```diff
@@ -3,5 +3,7 @@
 import android.content.ContentValues;
 
 public interface IAddDataEvents {
+
```
Comment (tigran-ita) [inline app/src/main/java/asus/example/com/mvpexample/IAddDataEvents.java]: Add new line

File: app/src/main/java/asus/example/com/mvpexample/Model.java [hunk 2]
```diff
@@ -20,25 +21,37 @@
...
+                        contentValues
+                );
             }
         };
         iLoadDataEvents = new ILoadDataEvents() {
             @Override
             public Cursor getloadDataCursor() {
-                return dbHelper.getReadableDatabase().query(DBHelper.DATA_TABLE_NAME, null,
-                        null, null, null, null, null);
+                return dbHelper.getReadableDatabase().query(
```
Comment (tigran-ita) [inline app/src/main/java/asus/example/com/mvpexample/Model.java]: Well done!

File: app/src/main/java/asus/example/com/mvpexample/IAddDataEvents.java (hunk 1 above)
Comment (SergeiMikhailovskii) [reply app/src/main/java/asus/example/com/mvpexample/IAddDataEvents.java]: done

File: app/src/main/java/asus/example/com/mvpexample/Model.java (hunk 2 above)
Comment (SergeiMikhailovskii) [reply app/src/main/java/asus/example/com/mvpexample/Model.java]: done

**Comment by SergeiMikhailovskii** [comment]
Ready for review

**Pull request merged**